climada/hazard/trop_cyclone.py

```diff
@@ -336,46 +336,93 @@ def from_tracks(
 
     def apply_climate_scenario_knu(
         self,
-        ref_year: int = 2050,
-        rcp_scenario: int = 45
+        percentile: str = '50',
+        rcp_scenario: int = 85,
+        ref_year: int = 2035,
+        baseline: tuple = (1982, 2022)
     ):
         """
-        From current TC hazard instance, return new hazard set with
-        future events for a given RCP scenario and year based on the
-        parametrized values derived from Table 3 in Knutson et al 2015.
-        https://doi.org/10.1175/JCLI-D-15-0129.1 . The scaling for different
-        years and RCP scenarios is obtained by linear interpolation.
-
-        Note: The parametrized values are derived from the overall changes
-        in statistical ensemble of tracks. Hence, this method should only be
-        applied to sufficiently large tropical cyclone event sets that
-        approximate the reference years 1981 - 2008 used in Knutson et. al.
-
-        The frequency and intensity changes are applied independently from
-        one another. The mean intensity factors can thus slightly deviate
-        from the Knutson value (deviation was found to be less than 1%
-        for default IBTrACS event sets 1980-2020 for each basin).
+        From current TC hazard instance, return new hazard set with future events
+        for a given RCP scenario and year based on the parametrized values derived
+        by Jewson 2021 (https://doi.org/10.1175/JAMC-D-21-0102.1) based on those
+        published by Knutson 2020 (https://doi.org/10.1175/BAMS-D-18-0194.1). The
+        scaling for different years and RCP scenarios is obtained by linear
+        interpolation.
+
+        Note: Only frequency changes are applied as suggested by Jewson 2022
+        (https://doi.org/10.1007/s00477-021-02142-6). Applying only frequency anyway
+        changes mean intensities and most importantly avoids possible inconsistencies
+        (including possible counting) that may arise from the application of both
+        frequency and intensity changes, as the relatioship between these two is non
+        trivial to resolve.
 
         Parameters
```

Comment:
I would describe the parameters in the same order as they appear in the function call.